spec/spec_helper_acceptance_local.rb

```diff
@@ -43,6 +43,10 @@ def update_profile_file
   run_shell("sed -i '/mesg n || true/c\\test -t 0 && mesg n || true' ~/.profile")
 end
 
+def get_os_name # rubocop:disable Style/AccessorMethodName: Do not prefix reader method names with get_
+  run_shell('facter os.name').stdout.delete("\n").downcase
```

Comment:
```suggestion
  @facter_os_name ||= run_shell('facter os.name').stdout.delete("\n").downcase
```

Add some caching (this won't change during the lifetime of the test runs).